examples/java-matter-controller/java/src/com/matter/controller/commands/common/MatterCommand.java

```diff
@@ -100,39 +103,22 @@ public void run() throws Exception {
 
   protected abstract void runCommand();
 
-  public void setTestResult(String result) {
-    mTestResult = Optional.of(result);
+  public void setSuccess() {
```

Comment:
How about `setResult(RealResult r)` and then callers would do `setResult(RealResult::Success())` or `setResult(RealResult::Failure("my reason"))` ?

Reply:
agree, done, thanks